### ISSUE TITLE
Alien::gmake v0.15 forces Alien::MSYS on Windows but not needed under MSYS2

### ISSUE THREAD

**Issue by zmughal** [Perl5-Alien/Alien-gmake#2]
We could add a patch to skip Alien::MSYS if it is not available.

**Comment by plicease** [comment]
It actually only pulls in Alien::MSYS if it can't find gmake already installed.

**Comment by plicease** [comment]
I believe this should be fixed in 0.15_01.  @zmughal let me know if this works for me.  @wbraswell please verify it doesn't break you.  If I hear back from you I can do a prod release real quick.

**Comment by zmughal** [comment]
Works for me!

```
zaki@MSI MINGW64 ~
$ cpanm Alien::gmake@0.15; cpanm Alien::gmake@0.15_01
--> Working on Alien::gmake
Fetching http://www.cpan.org/authors/id/P/PL/PLICEASE/Alien-gmake-0.15.tar.gz ... OK
Configuring Alien-gmake-0.15 ... OK
Building and testing Alien-gmake-0.15 ... ! Installing Alien::gmake failed. See C:\msys64\home\zaki\.cpanm\work\1484240622.16728\build.log for details. Retry with --force to force install it.
FAIL
! Finding Alien::gmake (== 0.15_01) on cpanmetadb failed.
--> Working on Alien::gmake
Fetching http://cpan.metacpan.org/authors/id/P/PL/PLICEASE/Alien-gmake-0.15_01.tar.gz ... OK
Configuring Alien-gmake-0.15_01 ... OK
Building and testing Alien-gmake-0.15_01 ... OK
Successfully installed Alien-gmake-0.15_01
1 distribution installed

zaki@MSI MINGW64 ~
$ perl -MAlien::MSYS -e1
Can't locate Alien/MSYS.pm in @INC (you may need to install the Alien::MSYS module) (@INC contains: C:/msys64/mingw64/lib/perl5/site_perl/MSWin32-x64-multi-thread C:/msys64/mingw64/lib/perl5/site_perl C:/msys64/mingw64/lib/perl5/vendor_perl C:/msys64/mingw64/lib/perl5/core_perl .).
BEGIN failed--compilation aborted.
```

**Comment by wbraswell** [comment]
Alien::astyle passes tests in both Travis-CI & AppVeyor with hard-coded use of Alien::gmake v0.15_01

https://travis-ci.org/wbraswell/alien-astyle/builds/191433529

https://ci.appveyor.com/project/wbraswell/alien-astyle/build/1.0.41

**Comment by plicease** [comment]
fixed by production release 0.16
